spicelib/scripts/asc_to_qsch.py

```diff
@@ -94,9 +94,10 @@ def convert_asc_to_qsch(asc_file, qsch_file, search_paths=[]):
         if symbol_tag is None:
             # Will try to get it from the sym folder
             print(f"Searching for symbol {comp.symbol}...")
+            # TODO: this should use the default locations from AscEditor, and use search_file_in_containers, just like AscEditor does.
```

Comment:
We already import the AscEditor into this space. Maybe we could "borrow" the code implemented on the AscEditor.
This could be done at a later stage.

Reply:
was hastitating to work on that, as I don't have qspice (yet), since it's windows only. Mark this for later?